build.sbt

```diff
@@ -36,8 +36,8 @@ lazy val extractJarSettings = Defaults.coreDefaultSettings ++ Seq(
 
 val commonSettings = Seq(
   organization := "com.gu",
-  scalaVersion := "2.12.3",
-	crossScalaVersions := Seq("2.11.11", scalaVersion.value),
+  scalaVersion := "2.12.8",
+	crossScalaVersions := Seq("2.11.12", scalaVersion.value),
```

Comment:
I believe this change may be causing errors when it works its way into Atom Workshop. I'm getting Play errors and this warning: `Warning:Cannot find project Scala library 2.11.8 for module atom-workshop` 
Any ideas if this is related?

Reply:
NB: Atom workshop is still on `scalaVersion := "2.11.8"`